src/Support/OMOptions.cpp

```diff
@@ -47,11 +47,3 @@ llvm::cl::opt<std::string> acceleratorTarget("maccel",
     llvm::cl::desc("Specify an accelerator to generate code for\n"
                    "\"NONE\" or \"\" for no accelerator\n"),
     llvm::cl::init(""), llvm::cl::cat(OMPassOptions));
-
-llvm::cl::opt<OptLevel> OptimizationLevel(
```

Comment:
Can we move the other options in this file as well ?

Reply:
I think we can do this, just a bit hesitate since I am not clear why this options were separately put here ...

Reply:
Done.